tensorrt_llm/runtime/production_debt.py

```
@@ -0,0 +1,207 @@
+        self,
+        never_equate_intent_to_approval: bool = True,
+        max_acceptable_tdi: float = 12.0,
+    ) -> None:
+        self.never_equate_intent_to_approval = never_equate_intent_to_approval
+        self.max_acceptable_tdi = max_acceptable_tdi
+        self.ledger = TechnicalDueDiligenceLedger()
+
+    def check_kill_switch(self) -> bool:
+        if os.environ.get("AAG_KILL_SWITCH", "").lower() in ("true", "1", "yes"):
+            return True
+        return any(Path(p).exists() for p in ("artifacts/KILL", "/tmp/KILL"))
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/KILL"

(S108)